services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -315,6 +315,7 @@ export class ProcessPermit extends Component {
       application_last_updated_date: noticeOfWork.last_updated_date
         ? formatDate(noticeOfWork.last_updated_date)
         : formatDate(noticeOfWork.submitted_date),
+        preamble_text: amendment.preamble_text,
     };
```

Comment:
I thought this was the only update needed on this file.

Reply:
it is, this is the only change.